### PR TITLE
improve ui

### PR DESCRIPTION
**Pull request by codebywinston**
- Save with CTRL-Enter instead of Enter
- Don't save on Escape
- Save on blur, when clicking outside of text area

### CODE REVIEW

File: static/flm.js [hunk 1]
```diff
@@ -1,123 +1,177 @@
...
-	       }
+    if (e.key == 'Escape') {
+      content.blur()
+      didJustEscape = true
     }
-    if (e.key == "Escape") {
-	content.blur();
+  })
+
+  console.log('hi aaron')
```
Comment (shapr) [inline static/flm.js]: :smile: 
Want to get rid of the debug code?

**Review by shapr — CHANGES_REQUESTED** [review]
Want to get rid of the debug console.log line?

File: static/flm.js (hunk 1 above)
Comment (codebywinston) [reply static/flm.js]: hi aaron

**Review by shapr — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**